resources/js/dashboard/pages/administrator/users/Create.vue

```diff
@@ -1,15 +1,121 @@
 <template>
-  <div>
-    create
+  <div class="main">
+    <Header :title="title" :iconClass="iconClass"/>
+    <b-form @submit="onSubmit" @reset="onReset" v-if="show" class="col-sm-10 col-md-6 mx-auto my-3 bg-light p-3" >
```

Comment:
Utilisons plus tot la directive `@submit.prevent` pour ne pas envoyer le formulaire en synchronone 
```suggestion
    <b-form @submit.prevent="onSubmit" @reset="onReset"  class="col-sm-10 col-md-6 mx-auto my-3 bg-light p-3" >
```

Reply:
pourquoi le `v-if="show"`

Reply:
> Bien recu